src/components/templates/SimpleTable/SimpleTableManual.tsx

```diff
@@ -334,6 +334,7 @@ export default function SimpleTableManual(props) {
                         <button
                           type='button'
                           className='psdk-utility-button'
+                          id='delete-button'
```

Comment:
Here also, I think you can assign different id's to these buttons that will allow us to identify `delete button` for a specific record/row. 
I've done something like that for Field Group.

Reply:
Oh okay, I was identifying the different row elements by nth value.